benchmarks/cpp/nvfuser/layer_norm.cpp

```diff
@@ -53,6 +53,68 @@ static void setupLayerNorm(Fusion* fusion, DataType dtype) {
   fusion->addOutput(output);
 }
 
+static void setupRMSNorm(Fusion* fusion, DataType dtype) {
+  TORCH_INTERNAL_ASSERT(dtype == DataType::Float || dtype == DataType::Half || dtype == DataType::BFloat16);
+
+  FusionGuard fg(fusion);
+
+  const int kReductionAxis = 2;
+  const float kEps = 1e-6;
+
+  Double* eps_ptr = IrBuilder::create<Double>(kEps);
+
+  // setup fusion
+  auto input = makeContigTensor(3, dtype);
+  auto weight = makeContigTensor(1, dtype);
+
+  fusion->addInput(input);
+  fusion->addInput(weight);
+
+  if (dtype == DataType::Half) {
+    input = castOp(DataType::Float, input);
+    weight = castOp(DataType::Float, weight);
+  }
+
+  auto rms_norm_results = rms_norm(input, 1, weight, eps_ptr);
+
+  auto output = rms_norm_results.output;
+
+  if (dtype == DataType::Half) {
```

Comment:
Shouldn't this be:
```.cpp
if (dtype != DataType::Float) {
  output = castOp(dtype, output);
}
  ```

Reply:
Also changed for the original layernorm functions